src/language-graphql/printer-graphql.js

```diff
@@ -1,15 +1,8 @@
 "use strict";
 
 const {
-  concat,
-  join,
-  hardline,
-  line,
-  softline,
-  group,
-  indent,
-  ifBreak,
-} = require("../document").builders;
+  builders: { concat, join, hardline, line, softline, group, indent, ifBreak },
+} = require("../document");
```

Comment:
I don't know why, but this change removes the TypeScript typecheck error. Why...?

Reply:
Maybe https://github.com/prettier/prettier/pull/9207